alacritty/src/renderer/mod.rs

```diff
@@ -378,8 +380,9 @@ impl GlyphCache {
         let padding_y = f64::from(window_config.padding.y) * dpr;
 
         // Calculate new size based on cols/lines specified in config.
-        let grid_width = cell_width as u32 * dimensions.columns_u32();
-        let grid_height = cell_height as u32 * dimensions.lines_u32();
+        // Override cols/lines if they are lower than the limit.
+        let grid_width = cell_width as u32 * max(dimensions.columns_u32(), MIN_COLS as u32);
+        let grid_height = cell_height as u32 * max(dimensions.lines_u32(), MIN_LINES as u32);
```

Comment:
Why are you limiting dimensions here? I don't think that will give you the correct results.

Reply:
This is where the window's dimensions are calculated from the config: https://github.com/alacritty/alacritty/blob/1bf40a8cc77d9082dfb76e028e0f66af88d79958/alacritty/src/display.rs#L174. 

Limiting it here is only effective on window creation however. I've added the limit to resizing.

Reply:
You should only change the requested dimensions though, not the resulting ones. We do not want to mess with the projection, the projection should still be correct.

Reply:
Disregard this comment, I think it should work out fine to do things here. The placement of the method is just really unfortunate since it's never used outside of display.rs. Since this just does the handling of the config's columns/lines and doesn't actually determine the size (it just requests a size), this is not a problem.